nautobot/extras/nautobot_app/__init__.py

```diff
@@ -0,0 +1,136 @@
+from django.apps import AppConfig
```

Comment:
Can this file be moved into `core`? It should be considered part of the core Nautobot functionality and I'd like to keep `extras` from continuing to be a dumping ground for new features.